### PR TITLE
HOSTEDCP-951: Let install apply to aggregate errors

### PR DESCRIPTION
**Pull request by enxebre**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: cmd/install/install.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/errors"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
```
Comment (muraee) [inline cmd/install/install.go:34]: duplicate import here.

**Comment by enxebre** [comment]
/hold
I want https://github.com/openshift/hypershift/pull/2374 first

**Pull request merged**